src/main/java/com/makersacademy/acebook/model/Post.java

```diff
@@ -16,12 +18,18 @@
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    private String content;
+    private String content_post;
+    private int user_id;
```

Comment:
Is there a reason why line 20 (`private Long id;`) has a `Long` datatype and line 22 (`private int user_id`) has an `int` datatype instead of them being the same?

This appears in all classes

Reply:
(the post one was set up like this 🙈 )